microcosm_flask/sentry.py

```diff
@@ -0,0 +1,67 @@
+from dataclasses import dataclass
+from os import environ
+from typing import Any, Optional
+from unittest.mock import MagicMock
+
+from microcosm.decorators import defaults
+from microcosm.object_graph import ObjectGraph
+from microcosm_logging.decorators import logger
+
+from sentry_sdk.utils import BadDsn
+
+
+@dataclass
+class SentryConfig:
+    dsn: Optional[str] = None
+    enabled: bool = False
+    client: Optional[Any] = None
```

Comment:
Can it be an actual client type from `sentry_sdk`?

Reply:
How the client is exposed in `sentry_sdk` is quite weird it goes through a few layers of lamdas which didn't play nice when I tried to add typing info